qemu/tests/cfg/nic_opt.cfg

```diff
@@ -0,0 +1,77 @@
+- nic_opt:
+    type = nic_opt
+    only virtio_net
+    only x86_64 i386 ppc64 ppc64le
+    Windows:
+        netperf_server_link_win = netserver-2.6.0.exe
+        netperf_client_link_win = netperf.exe
+        server_path_win = "c:\\"
+        client_path_win = "c:\\"
+        test_protocols = TCP_STREAM UDP_STREAM
+        netperf_package_sizes = 32 64 128 256 512 1024 2048 4096 8192 16384 32768
+    variants:
+        - opt_test_netperf:
+            netperf_link = netperf-2.6.0.tar.bz2
+            server_path = /var/tmp/
+            client_path = /var/tmp/
+            netperf_test_duration = 180
+            netperf_output_unit = m
+            netperf_local_cpu = yes
+            netperf_remote_cpu = yes
+            netperf_para_sessions = 1
+            test_protocols = TCP_STREAM
+            netperf_client = ${main_vm}
+            netperf_server = localhost
+            netperf_package_sizes = 1024
+            sub_type = netperf_stress
+            sub_exit_timeout = 10
+            vhost = on
+            variants:
+                - indirect_desc:
+                    check_type = netperf
+                        variants:
+                            - opt_on:
+                                nic_extra_params += ",indirect_desc=on"
+                            - opt_off:
+                                nic_extra_params += ",indirect_desc=off"
+                - mrg_rxbuf:
+                    check_type = netperf
+                        variants:
+                            - opt_on:
+                                nic_extra_params += ",mrg_rxbuf=on"
+                            - opt_off:
+                                nic_extra_params += ",mrg_rxbuf=off"
+        - opt_test_ping:
+            ping_package_sizes = 0 1 48 64 512 1440 1500 1505 4096 4192 32767 65500
+            variants:
+                - poll-us:
+                    check_type = ping
+                    queues = 4
+                    vectors = 10
+                    variants:
+                        - opt_30:
+                            netdev_extra_params += ",poll-us=30"
+                        - opt_50:
+                            netdev_extra_params += ",poll-us=50"
+                - mqcontrol:
+                    check_type = ping
+                    variants:
```

Comment:
for mq control cases, we should test with queues=4, so that the "mq=on" is effective.

Reply:
I have changed the file content, now the option "mq=on" is effective.

Reply:
> I have changed the file content, now the option "mq=on" is effective.

no need to add vectors=10, it will assign 10 vectors automatically when setting "queues=4"

Reply:
I have deleted "vectors=10".